docs/application/toc_all.md

```diff
@@ -16,6 +16,13 @@
 ## [Overview](/application/dotnet/index.md)
 ## [Learn about .NET](/application/dotnet/learn-about-dotnet.md)
 
+## Get Started using VSCode
+### [.NET application development](/application/vscode-ext/Tizen/dotnet.md)
+
+## Get Started using VS
+### [.NET application development](/application/vstools/Tizen/dotnet.md)
```

Comment:
> One concern, @mijin85cho suggested we merge the the content of vscode-ext/Tizen/dotnet.md and vstools/Tizen/dotnet.md into dotnet/get-started/first-app.md.

- **About `/application/vstools/Tizen/dotnet.md`** : 
   This page and dotnet/get-started/first-app.md are following almost same steps. 
   Only the last section, "Debug your application in emulator" is the difference. In my thought merging the Debug section to  dotnet/get-started/first-app.md will be enough. Would you please let me know your opinion?

- **About /application/vscode-ext/Tizen/dotnet.md** : 
   This page and dotnet/get-started/first-app.md are not in same steps. 
   It is giving more description on how to install Visual Studio Code Extension for Tizen & troubleshooting.
   Hmm... In my thought comparing dotnet/get-started/first-app/ and /application/vscode-ext/Tizen/dotnet.md to enhance the create & build & deploy sections in /application/vscode-ext/Tizen/dotnet.md, and leaving other parts as basic description for how to install Visual Studio Code Extension for Tizen can be an idea, but not sure. I'd like to know your thought about it.

Reply:
Agreed with @mijin85cho 's direction 👍 

Thank you!

Reply:
@mijin85cho hello!

I have gone through your observations and I agree with you on both points. 

For the first point, “About /application/vstools/Tizen/dotnet.md :”, I also think the “Debug your application in emulator” sections for vscode extension (application/vscode-ext/Tizen/dotnet.md) and CLI (if applicable) should also be added to the dotnet/get-started/first-app.md file, as the Create, Build, and Deploy section contains content for all 3 (Visual Studio 2019, VS Code, and CLI).

For the second point, “About /application/vscode-ext/Tizen/dotnet.md :”, I have also observed that there is a difference between the contents of the files dotnet/get-started/first-app.md and application/vscode-ext/Tizen/dotnet.md. You have suggested to enhance the create & build & deploy sections in /application/vscode-ext/Tizen/dotnet.md by taking content from dotnet/get-started/first-app.md. I believe that is the right step to take and agree with your decision.

Reply:
> For the second point, “About /application/vscode-ext/Tizen/dotnet.md :”, I have also observed that there is a difference between the contents of the files dotnet/get-started/first-app.md and application/vscode-ext/Tizen/dotnet.md. You have suggested to enhance the create & build & deploy sections in /application/vscode-ext/Tizen/dotnet.md by taking content from dotnet/get-started/first-app.md. I believe that is the right step to take and agree with your decision.

Thanks for the reply, @asm-foysal  : )

Just one point, I think my sentence was not clear. My intention was to take create & build & deploy sections from /application/vscode-ext/Tizen/dotnet.md and enhance the dotnet/get-started/first-app.md. Other "Installing VS Code Extension" parts will be remain as /application/vscode-ext/Tizen/dotnet.md, under /application/vscode-ext/. Sorry for the confusion.

Would you please create another PR doing that? We'll check that PR and give you comments.

Reply:
@mijin85cho thank you for the clarification, however, after going through both the files, it seems the create & build & deploy sections in the file **dotnet/get-started/first-app.md** is a lot more detailed than the same sections in **application/vscode-ext/Tizen/dotnet.md**. I'm just not sure how this enhancement would work...

Reply:
@asm-foysal I see, thank you for letting me know your concern. 

I also know that **dotnet/get-started/first-app.md** is much more rich in content. 
So my meaning to "enhance **dotnet/get-started/first-app.md**" was to add note content _"If you have a Tizen device and want to deploy the application to it, you must create a certificate profile using the Tizen [Certificate Manager](https://docs.tizen.org/application/vstools/tools/certificate-manager) or Tizen CLI before building your project. For more information, see Certificate Manager."_ to **dotnet/get-started/first-app.md** , or adding a sentence _"You cannot launch an emulator directly from Visual Studio Code. Instead, you must launch the Tizen Emulator Manager and use it to launch the emulator you need."_ to to **dotnet/get-started/first-app.md**, etc.
I also think there would not be so much reinforcing. Maybe you can think in other way, as like trimming **application/vscode-ext/Tizen/dotnet.md** to make it fit under "Tizen Tools Family / Visual Studio Code Extension for Tizen / " (same level with "prerequisites", after it)